File: Sesion 6/Marco teorico/1QUÉ ES UN ARREGLO DE OBJETOS/1.py
class Persona:
    def __init__(self, nombre, edad, genero):
        self.nombre = nombre
        self.edad = edad
        self.genero = genero

    def mostrar_datos(self):
        print("Nombre:", self.nombre, ", Edad:", self.edad, ", Género:", self.genero)



personas = [
    Persona("Juan", 25, 'M'),
    Persona("María", 30, 'F'),
    Persona("Pedro", 40, 'M')
]

for persona in personas:
    persona.mostrar_datos()

File: Sesion 6/Marco teorico/1QUÉ ES UN ARREGLO DE OBJETOS/1.java
class Persona {
    private String nombre;
    private int edad;
    private char genero;

    public Persona(String n, int e, char g) {
        this.nombre = n;
        this.edad = e;
        this.genero = g;
    }

    public void mostrarDatos() {
        System.out.println("Nombre: " + nombre + ", Edad: " + edad + ", Género: " + genero);
    }
}

public class Main {
    public static void main(String[] args) {
        Persona[] personas = {
            new Persona("Juan", 25, 'M'),
            new Persona("María", 30, 'F'),
            new Persona("Pedro", 40, 'M')
        };

        for (Persona persona : personas) {
            persona.mostrarDatos();
        }
    }
}
